cleanup-oracle.sh

```diff
@@ -55,13 +72,33 @@ while true; do
         shift;
         ;;
     --inventory-file)
-	INVENTORY_FILE="$2"
+      	INVENTORY_FILE="$2"
```

Comment:
Spaing looks a bit off here, and on the next line too.

Reply:
removed and added spaces back